jena/src/main/scala/org/w3/banana/jena/io/JenaRDFReader.scala

```diff
@@ -23,7 +23,7 @@ final class TripleSink(implicit ops: JenaOps) extends StreamRDF {
   def triple(triple: JenaTriple): Unit = {
     def isXsdString(node: JenaNode): Boolean =
       node.isLiteral &&
```

Comment:
Although not originating from this PR, the body must be enclosed in braces as it is not a single line.